projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid.component.ts

```diff
@@ -360,12 +362,80 @@ export class IgxTreeGridComponent extends IgxGridBaseDirective implements GridTy
             this.loadChildrenOnRowExpansion(args);
         });
 
+        this.onRowAdded.subscribe(args => {
```

Comment:
Don't leave the args as `any`, the event emitter is typed but that's not picked up when you don't use a pipe. Then again, you might as well use one too :)